interfaces/Portalicious/src/app/app.routes.ts

```diff
@@ -70,7 +76,29 @@ export const routes: Routes = [
       },
       {
         path: AppRoutes.projectRegistrations,
-        component: ProjectRegistrationsComponent,
+        children: [
+          {
+            path: ``,
+            component: ProjectRegistrationsComponent,
+          },
```

Comment:
Refactor this to `registrations` vs `registration/:id` (similarly to `projects` vs `project/:id`)